charts/openebs/templates/kyverno/allow-host-namespaces.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: allow-host-namespaces
+  namespace: openebs
+  annotations:
+    policies.kyverno.io/category: Pod Security Standards
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      Host namespaces (Process ID namespace, Inter-Process Communication namespace, and
+      network namespace) allow access to shared information and can be used to elevate
+      privileges. Pods should  allowed access to host namespaces.      
+spec:
+  validationFailureAction: audit
+  background: true
+  rules:
+    - name: host-namespaces
+      match:
+        resources:
+          name: "openebs-*"
```

Comment:
q_: if openebs is installed in a different namespace than `openebs`, will this still work? Can we have a check in based on `openebs.io/component-name`

Reply:
We have to use the selector(matchLabels) for example: https://kyverno.io/docs/writing-policies/match-exclude/#advanced-match-statement

Reply:
If we remove these three `kind: ClusterPolicy`  `namespace: openebs`  `name: "openebs-*"`  then the policies will work in other namespaces. And  `Policy `instead of `ClusterPolicy`  then namespaced policies will only apply to resources within the namespace in which they are defined.